src/main/java/org/jclouds/vcloud/director/v1_5/compute/strategy/VCloudDirectorComputeServiceAdapter.java

```diff
@@ -676,16 +675,21 @@ private Vdc getVdc(String locationId) {
    }
 
    private Set<Vm> getAvailableVMsFromVAppTemplate(VAppTemplate vAppTemplate) {
-      return ImmutableSet.copyOf(filter(vAppTemplate.getChildren(), new Predicate<Vm>() {
-         // filter out vms in the vApp template with computer name that contains underscores, dots,
-         // or both.
-         @Override
-         public boolean apply(Vm input) {
-            GuestCustomizationSection guestCustomizationSection = api.getVmApi().getGuestCustomizationSection(input.getId());
-            String computerName = guestCustomizationSection.getComputerName();
-            return computerName.equals(computerName);
-         }
-      }));
+      // FIXME What should this be? Previously it did `computerName.equals(computerName)`,
```

Comment:
@andreaturli It feels like all VMs in the vApp template should be "available". We shouldn't be filtering any out at this stage. I'm therefore still tempted to delete this filter (which is effectively preserving the old behaviour).